### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by fortuneonyeka**
In this milestone, I implemented the following :
- Initialise React app.
- Structure application files using a "feature folder" approach and the duck's pattern for  Redux files.
- Add React Router and set two <Route>s and <Link>s for the app's navigation.
- Display the list of books.
- Have a form for adding a book 
- Categories displaying under construction" text only.

### CODE REVIEW

File: src/redux/books/initialbookstate.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  // {
+  //   author: 'Unknown',
+  //   title: 'Around the globe',
+  //   id: 5,
+  // },
+  // {
+  //   author: 'Unknown',
+  //   title: 'One Thousand and One Nights',
+  //   id: 6,
+  // },
```
Comment (rukundoeric) [inline src/redux/books/initialbookstate.js]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @fortuneonyeka, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/redux/books/initialbookstate.js (hunk 1 above)
Comment (rloterh) [reply src/redux/books/initialbookstate.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Fortune, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**